README.md

```
@@ -44,10 +44,26 @@ python3 -m pip install databusclient
+#   --help                                               Show this message and exit.
+#
+# Commands:
+#   deploy
+#   download
+#   delete
+#   mkdist
+#   completion
+```
+
+### Download command
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)